tests/functionality/sopflow/selfcheck.cpp

```diff
@@ -431,7 +438,15 @@ int main(int argc, char **argv) {
   ExaGOCheckError(ierr);
   ExaGOLog(EXAGO_LOG_INFO, "{}", "Creating SOPFLOW Functionality Test");
 
-  SopflowFunctionalityTests test{std::string(argv[1])};
+  int my_rank;
+  auto err = MPI_Comm_rank(comm, &my_rank);
```

Comment:
Still some refactoring to do in other application tests to refactor out this rank stuff.

Reply:
This occurs outside of FunctionalityTestContext which is initialized later in the code. I'd just leave this one (there is similar code in the `main` routines in all the other `selfcheck.cpp` codes).